python/pyarrow/pandas_compat.py

```diff
@@ -371,8 +371,8 @@ def _get_columns_to_convert(df, schema, preserve_index, columns):
     # to be converted to Arrow format
     # columns_types : specified column types to use for coercion / casting
     # during serialization, if a Schema was provided
-    return (all_names, column_names, index_descriptors, index_levels,
-            columns_to_convert, convert_types)
+    return (all_names, column_names, index_column_names, index_descriptors,
```

Comment:
> At this point, perhaps we should return a namedtuple?

Since it is only used in two places close by in the code in this file, I have the feeling it wouldn't necessarily add much value